### DISCUSSION TITLE
Retrieve com.typesafe.config.Config from CommentedConfigurationNode

### DISCUSSION THREAD

**Discussion by yiannis-zen** [SpongePowered/Configurate#317] [Support]
Hi there! awesome library, very useful for working with the different configuration formats.
Was wondering if there is a way to retrieve com.typesafe.config.Config from a CommentedConfigurationNode
Thanks again!

**Comment by zml2008** [comment]
I'm afraid not - we consider the backing library for most loaders to be an implementation detail, so specific classes are not exposed.

It would likely be possible to write a TypeSerializer to create a Config object from a node, if you wanted, but that would take some work.

**Reply by yiannis-zen** [reply]
No worries at all! That was my understanding, just wanted to double-check.
Would be useful for my use-case since I am working with code heavily dependent on the com.typesafe.config.* API but I can definitely live without it :)
